File: beecrowd_exercicios_em_Cpp/beginner/calculando.cpp
#include <iostream>

using namespace std;

int main(){

    int n;

    cin >> n;

    for(int i = 0; i < n; i ++){

        string operacao;

        cin >> operacao;

        int rslt = operacao.at(1) - '0';

        for(int j = 1; j < operacao.size(); j +=2){

        }

        cout << "Teste "<< rslt << endl;
    }

    return 0;
}

File: beecrowd_exercicios_em_python/begginner/calculando.py
n = int(input())
j = 1

while n != 0:

    op = input()

    rslt = 0
    vals = {"+": 1, "-": -1, op[-1]: 1}
    sin = 1
    num = ""

    for i in range(len(op)):

        if op[i] == "+" or op[i] == "-":
            rslt += int(num) * sin
            sin = vals[op[i]]
            num = ""

        else:
            num += op[i]

    print("Teste {:d}\n{:d}\n".format(j, rslt + int(num)*sin))

    n = int(input())

    j += 1
